src/core_modules/capture-core/components/WidgetsRelationship/WidgetTrackedEntityRelationship/NewTrackedEntityRelationship/hooks/useAddRelationship.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+import i18n from '@dhis2/d2-i18n';
+// $FlowFixMe - useAlert is exported from app-runtime
```

Comment:
Did not know we could edit those, thanks for the tip!

However, I see that we already export `any`;
```js
declare module '@dhis2/app-runtime' {
  declare module.exports: any;
}
```

Should this not take care of `useAlerts` as well? Asking out of pure curiosity, don't mind defining the hook explicitly like your suggestion.

Reply:
As far as I understand, if `@dhis2/app-runtime` used `flow`, then yes, we could automatically export and use its definitions. But since they don't use it, `declare module.exports: any` doesn't really do much and we have to manually define what we import and use.

Reply:
@eirikhaugstulen The `any` declaration is overridden in `src/core_modules/capture-core/flow/app-runtime_v2.x.x.js`. So adding a definition there would be great.